presto-docs/src/main/sphinx/release/release-0.284.rst

```diff
@@ -0,0 +1,67 @@
+=============
+Release 0.284
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix correctness bug in the optimizer when ``randomize_outer_join_null_key`` is enabled.
+* Fix bug in array ``CONTAINS`` expression rewrite rule.
+* Fix bugs parsing lambda expressions.  The bugs may cause compile exception with message "Can not compile special form: WHEN" and lead to runtime exception of array out of bound access.
+* Improve cache efficiency of history-based optimizer.
+* Add field names when casting row to JSON.
+* Add :func:`array_top_n` to return an array of top ``N`` elements of a given array.
+* Add session property ``restrict_history_based_optimization_to_complex_query``. When set to ``TRUE``, only queries with join or aggregation will try to use HBO. The default value is ``TRUE``.
+* Add :func:`bitwise_xor_agg` function.
+* Add session property ``pull_expression_from_lambda_enabled`` to optimize lambda functions which have expressions not referring to arguments of the lambda function.  The default state is enabled using the value ``TRUE``.
+* Add ``rewrite_constant_array_contains_to_in_expression`` session property to improve the performance of ``CONTAINS`` expressions.  The default state is enabled using the value ``TRUE``.
+* Improve performance of outer join when nulls are present. It can be enabled by setting ``randomize_outer_join_null_key_strategy`` to be ``cost_based``. The trigger condition can be set by session properties ``randomize_outer_join_null_key_null_count_threshold`` and ``randomize_outer_join_null_key_null_ratio_threshold``.
+* Add function :func:`trail`.
+* Add session property ``optimizers_to_enable_verbose_runtime_stats`` to enable runtime tracking for a set of optimizers.
+* Add support for ``EXPLAIN (TYPE VALIDATE)`` of ``EXPLAIN`` queries. Previously such queries would fail with an error.
+* Improve performance of cluster statistics reporting in the Presto coordinator by adding the configuration property ``cluster-stats-cache-expiration-duration``. The property is ``0`` (disabled) by default.
+* Add support for serializing nested data structures via server-side configuration properties using the configuration property ``nested-data-serialization-enabled``. The property is enabled by default with the value ``TRUE``.
+* Add a Redis-based historical statistics provider.  (See :doc:`Redis HBO Provider Documentation </plugin/redis-hbo-provider>`).
```

Comment:
```suggestion
* Add a Redis-based historical statistics provider.  For more information, see :doc:`Redis HBO Provider </plugin/redis-hbo-provider>`.
```
Nit edit to remove unneeded parentheses, and to remove "Documentation" from the link text as it's not part of the destination title. Considered and rejected moving "documentation" to after the link - left it out entirely as I didn't feel the word added value for the reader.

Reply:
Done